java/src/Ice/src/main/java/com/zeroc/Ice/Current.java

```diff
@@ -256,15 +254,15 @@ private OutgoingResponse createOutgoingResponseCore(Throwable exc) {
     } else if (exc instanceof UnknownLocalException ex) {
       exceptionId = ex.ice_id();
       replyStatus = ReplyStatus.UnknownLocalException;
-      unknownExceptionMessage = ex.unknown;
+      unknownExceptionMessage = ex.toString();
```

Comment:
This is not correct. The message of an UnknownXxxException is returned by getMessage, not toString.

Reply:
Also fixed.